modules/strelka/1.1/strelka.smk

```diff
@@ -246,7 +248,7 @@ rule _combine_strelka:
         stderr = CFG["logs"]["strelka"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/strelka_combine.stderr.log"
     shell:
         op.as_one_line("""
-        bcftools concat -a {input.snv} {input.indel} | bcftools sort --max-mem {resources.mem_mb}M -Oz -o {output.combined} > {log.stdout} 2> {log.stderr} && tabix -p vcf {output.combined} >> {log.stdout} 2>> {log.stderr} 
+        bcftools concat -a {input.vcf} | bcftools sort --max-mem {resources.mem_mb}M -Oz -o {output.combined} > {log.stdout} 2> {log.stderr} && tabix -p vcf {output.combined} >> {log.stdout} 2>> {log.stderr} 
```

Comment:
Is "-p vcf" not required? I noticed you didn't use it in gridss when running tabix on your vcfs.